smoke-tests/README.md

```diff
@@ -0,0 +1,36 @@
+# Smoke tests
+
+## How to run smoke tests with docker
+
+1. Build the docker image:
+```bash
+docker build -t smoke-tests .
+```
+
+2. Run the docker container:
+```bash
+docker run  -it -v "$(pwd):/usr/src/app" smoke-tests
+```
+
+## How to run smoke tests without docker
+1. Install the dependencies:
+```bash
+npm install
+```
+
+2. Configure the environment variables:
+```bash
+cp .env.dist .env
+```
+
+3. Run the tests with a visual interface:
+```bash
+npm run test:ui # With a visual interface
+npm run test # Without a visual interface
+```
+
+## How to write smoke tests
```

Comment:
FYI, that's something I haven't tried.

Reply:
Yes, it was more documented in case you wanted to add quickly other tests.